examples/expressions_example/.i18nrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "prefix": "expressionsExample",
+  "paths": {
+    "expressionsExample": "."
+  },
+  "translations": ["translations/ja-JP.json"]
```

Comment:
nit: probably can remove this. i know this scaffolded on generation

Reply:
I wasn't sure if we were going to use these files for the localization that we are reintroducing. Figured if we were, its easier to start from this boilerplate, and if not we can just nuke this and all other `.i18nrc.json` files